multiapps-controller-process/src/test/java/org/cloudfoundry/multiapps/controller/process/steps/CreateOrUpdateStepWithExistingAppTest.java

```diff
@@ -84,10 +105,24 @@ static Stream<Arguments> testHandleStagingApplicationAttributes() {
     void testHandleStagingApplicationAttributes(Staging existingStaging, Staging staging, boolean expectedPropertiesChanged) {
         CloudApplication existingApplication = getApplicationBuilder(false).staging(existingStaging)
                                                                            .build();
+        if (staging.getCommand() == null) {
+            staging = ImmutableStaging.copyOf(staging)
```

Comment:
I have checked http://v3-apidocs.cloudfoundry.org/version/3.123.0/#processes and `command` field can be null, so is it necessary to set default one?